tools/d2g/d2g.go

```diff
@@ -107,11 +108,6 @@ func Scopes(dwScopes []string, dwPayload *dockerworker.DockerWorkerPayload, task
 				"generic-worker:os-group:"+taskQueueID+"/kvm",
 				"generic-worker:os-group:"+taskQueueID+"/libvirt",
 			)
-		case s == "docker-worker:capability:privileged":
-			gwScopes = append(
-				gwScopes,
-				"generic-worker:os-group:"+taskQueueID+"/docker",
-			)
```

Comment:
Note, tasks will now get scope `generic-worker:capability:privileged` added, which does no harm, but doesn't help a lot. That said, there are other scopes it will also not translate, so that might be fine for now, and helps show what the original task had.